src/sentry/static/sentry/app/views/alerts/details/chart.tsx

```diff
@@ -3,13 +3,24 @@ import moment from 'moment';
 
 import {t} from 'app/locale';
 import space from 'app/styles/space';
+import theme from 'app/utils/theme';
+import {Trigger} from 'app/views/settings/incidentRules/types';
 import LineChart from 'app/components/charts/lineChart';
 import MarkPoint from 'app/components/charts/components/markPoint';
+import MarkLine from 'app/components/charts/components/markLine';
+import VisualMap from 'app/components/charts/components/visualMap';
 
 import closedSymbol from './closedSymbol';
 import detectedSymbol from './detectedSymbol';
 
 type Data = [number, {count: number}[]];
+type PiecesObject = {
```

Comment:
Can you pull this out of echarts options as well?

Reply:
We can't TypeScript doesn't like `EChartOption.VisualMap['PiecesObject']`

Reply:
What does it complain about?

Reply:
It says `Property 'PiecesObject' does not exist on type 'VisualMap'.` even though it does